saos-search/src/test/java/pl/edu/icm/saos/search/search/service/JudgmentCriteriaTransformerTest.java

```diff
@@ -40,13 +40,18 @@ public void setUp() {
     public static Object[][] criterionData() {
         return new Object[][] {
                 { "+content:word", new JudgmentCriteria("word") },
+                { "+(content:word1 content:word2)", new JudgmentCriteria("word1 or word2") },
                 { "*:*", new JudgmentCriteria(" ") },
                 
-                { "+judgeName:Adam\\ Nowak", new JudgmentCriteriaBuilder().withJudgeName("Adam Nowak").build() },
```

Comment:
I would leave this case, I mean Adam Nowak without double-quotes, becuase we don't have such a case below.